src/scss/components/optional/_overflow-carousel.scss

```diff
@@ -22,23 +36,33 @@
 
     .c-overflowCarousel-inner {
         @include media('<mid') {
-            overflow-x: scroll;
-            overflow-y: hidden;
-            -webkit-overflow-scrolling: touch;
-            width: 100%;
+            @include overflowCarouselInner();
 
             &.is-active {
                 overflow: visible;
             }
         }
+
+        .c-overflowCarousel--fromMid & {
+            @include media('<=mid') {
```

Comment:
Should it here and on line 63 be `<mid`?

Reply:
The fromMid modifier Includes the mid breakpoint - the others are below